ios/purchases_flutter.podspec

```diff
@@ -15,8 +15,7 @@ Pod::Spec.new do |s|
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
-  s.dependency 'Purchases', '~> 3.2.2'
-
+  s.dependency 'PurchasesHybridCommon', '1.1.0'
```

Comment:
this is how dependencies will be managed from now on, `PurchasesHybridCommon` already has a pointer to the `Purchases` version